gammapy/maps/wcs/tests/test_ndmap.py

```diff
@@ -639,6 +639,30 @@ def test_convolve_nd():
     assert_allclose(values_full, values_same, rtol=1e-5)
 
 
+def test_get_psf_kernel_multiscale():
```

Comment:
I find strange that this test in within gammapy.maps rather than gammpay.irf. 
It tests `PSFMap` and `PSFKernelMap`

Reply:
Is it really related to this PR?

Reply:
forgot to move the new tests into #5314

Reply:
moved